### PR TITLE
Fix admin views

### PR DESCRIPTION
**Pull request by DanielePalombo**
Fix the view for work with solidus 1.3 in related tab under edit product.

### CODE REVIEW

File: app/views/spree/admin/products/related.html.erb [hunk 1]
```diff
@@ -12,33 +12,33 @@
...
       <legend><%= Spree.t(:add_related_product) %></legend>
       <div class="row">
-        <div id="related_product_name" class="">
-          <div class="form-group">
-            <%= label_tag :add_variant_name, Spree.t(:name_or_sku_short) %>
-            <%= hidden_field_tag :add_variant_name, '', class: 'variant_autocomplete', style: "display: block !important;" %>
-          </div>
-        </div>
+        <div class="alpha four columns">
+					<div class="field">
```
Comment (graygilmore) [inline app/views/spree/admin/products/related.html.erb]: Can we switch these tabs to spaces? Prefer to keep the files consistent.

File: app/views/spree/admin/products/related.html.erb [hunk 2]
```diff
@@ -12,33 +12,33 @@
     <fieldset>
       <legend><%= Spree.t(:add_related_product) %></legend>
       <div class="row">
-        <div id="related_product_name" class="">
-          <div class="form-group">
-            <%= label_tag :add_variant_name, Spree.t(:name_or_sku_short) %>
-            <%= hidden_field_tag :add_variant_name, '', class: 'variant_autocomplete', style: "display: block !important;" %>
-          </div>
-        </div>
+        <div class="alpha four columns">
```
Comment (graygilmore) [inline app/views/spree/admin/products/related.html.erb]: We're moving away from classnames like this towards using some of the Bootstrap stuff that was implemented a while ago. Unfortunately, the Bootstrap classes aren't available to us yet. Maybe we could add a stylesheet with some styles like this: https://github.com/solidusio/solidus/blob/master/backend/app/assets/stylesheets/spree/backend/sections/_products.scss#L1-L16

File: app/views/spree/admin/products/related.html.erb [hunk 3]
```diff
@@ -12,33 +12,33 @@
...
         </div>
-
       </div>
-      <div class='row'>
-        <div class='column'>
-          <%= button_link_to Spree.t(:add), admin_product_relations_url(@product), icon: 'add', id: 'add_related_product', data: { update: 'products-table-wrapper' } %>
-        <div>
-      <div>
+
+			<div class="clearfix"></div>
```
Comment (graygilmore) [inline app/views/spree/admin/products/related.html.erb]: Instead of adding an empty `clearfix`

Comment (graygilmore) [inline app/views/spree/admin/products/related.html.erb]: So it'd be `class="row clearfix"`. That *should* work.

**Comment by DanielePalombo** [comment]
@graygilmore i made some changes after our conversation in slack. I hope now the commits are ready for merge.

**Pull request merged**